Resources/views/Collector/db.html.twig

```diff
@@ -98,289 +107,332 @@
         .time-bar { display: block; position: absolute; top: 0; left: 0; bottom: 0; background: #e0e0e0; }
         .sql-runnable.sf-toggle-content.sf-toggle-visible { display: flex; flex-direction: column; }
         .sql-runnable button { align-self: end; }
+        table .sort-icon { display: inline-flex; vertical-align: sub; }
+        table .sort-icon svg { height: 20px; width: 20px; margin-left: 2px; stroke-width: 2.5; color: var(--color-link); }
     </style>
 
-    <h2>Query Metrics</h2>
+    <h2>Doctrine</h2>
 
     <div class="metrics">
-        <div class="metric">
-            <span class="value">{{ collector.querycount }}</span>
-            <span class="label">Database Queries</span>
-        </div>
-
-        <div class="metric">
-            <span class="value">{{ collector.groupedQueryCount }}</span>
-            <span class="label">Different statements</span>
-        </div>
+        <div class="metric-group">
+            <div class="metric">
+                <span class="value">{{ collector.querycount }}</span>
+                <span class="label">Queries</span>
+            </div>
 
-        <div class="metric">
-            <span class="value">{{ '%0.2f'|format(collector.time * 1000) }} ms</span>
-            <span class="label">Query time</span>
-        </div>
+            <div class="metric">
+                <span class="value">{{ collector.groupedQueryCount }}</span>
+                <span class="label">Statements</span>
```

Comment:
The previous label was `Different statements`, and the key thing to distinguish it from the previous number is the word `Different` that you removed.